CHANGELOG.md

```diff
@@ -4,6 +4,51 @@
 
 ### Notable changes
 
+* **V8**: upgrade to 4.2.77.18, see the [ChangeLog](https://chromium.googlesource.com/v8/v8/+/refs/heads/4.2.77/ChangeLog) for full details. Notable items:
+  - Classes have moved out of staging, the `class` keyword is now usable without flags
```

Comment:
OK, did some experimenting and answered my own question `console.log({woo(){return "woot!"}}.woo())`, nice. I'd still appreciate some words from you on this @domenic if you don't mind.